quickstarts/install-odf.yaml

```diff
@@ -1,31 +1,31 @@
 apiVersion: console.openshift.io/v1
 kind: ConsoleQuickStart
 metadata:
-  name: ocs-install-tour
+  name: odf-install-tour
```

Comment:
I don't believe anything will delete the old `ocs-install-tour` quick start if you change the name. The simplest would be to keep the old `metadata.name` and just update the to reference ODF. (Only the display name should be shown in the UI.)

Reply:
Makes sense. Ideally I thought the operator may be reconciling to see which resources were available and deploying them.

Reply:
I've added the annotation to the older manifest.